extensions/adobe/experience/decisioning/option-detail.schema.json

```diff
@@ -85,6 +85,21 @@
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "The number of times a particular option was proposed so far to a particular profile. When this metric is present the record or time-series entity must include an IdentityMap the profile.",
           "meta:descriptionId": "option-detail##https://ns.adobe.com/experience/decisioning/propositionsProfile##description##18831"
+        },
+        "xdm:selectionId": {
```

Comment:
Please capitalize the `Id` to `ID` making the field `xdm:selectionID` as per the contributing guide: https://github.com/adobe/xdm/blob/master/CONTRIBUTING.md#coding-styleguides

Reply:
Thanks for pointing that out. It has been updated.